cmake/Modules/FindoneMKL.cmake

```diff
@@ -0,0 +1,24 @@
+set(ONEMKL_LIBRARIES)
+
+if(DEFINED ENV{MKLROOT})
+  SET(INTEL_MKL_DIR $ENV{MKLROOT})
+else()
+  SET(INTEL_MKL_DIR "/opt/intel/oneapi/mkl/latest")
+endif()
+
+set(CMAKE_INCLUDE_PATH ${CMAKE_INCLUDE_PATH}
+    "${INTEL_MKL_DIR}/include")
+set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH}
+    "${INTEL_MKL_DIR}/lib")
```

Comment:
Please use find_file to add a check whether there is a directory for `lib` or `include`.
<img width="298" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/37255121-96ee-4973-be71-ca44d9688d2e">